UnleashClient/__init__.py

```diff
@@ -185,7 +184,32 @@ def __init__(
 
         self.metrics_headers: dict = {}
 
-        # Scheduler bootstrapping
+        self._init_scheduler(scheduler, scheduler_executor)
+
+        if custom_strategies:
+            self.engine.register_custom_strategies(custom_strategies)
+
+        self.strategy_mapping = {**custom_strategies}
+
+        # Client status
+        self.is_initialized = False
+
+        # Bootstrapping
+        if self.unleash_bootstrapped:
+            OfflineConnector(
```

Comment:
Okay that's pretty elegant! I'm tempted to call this connector BootstrapConnector, just based on this. That's really what it's doing. Fine if you like OfflineConnector though